src/pages/developers/omnichain/tutorials/pyth.mdx

```diff
@@ -37,3 +37,26 @@ ensuring that the price feeds are accurate and up-to-date.
 Currently supported assets: BTC, BNB, MATIC, ETH, USDC, ZETA, USDT.
 
 Price feed IDs: https://pyth.network/developers/price-feed-ids
+
+# Pyth Entropy
+
+[Pyth Entropy] (https://docs.pyth.network/entropy) allows developers to quickly and easily generate secure 
+random numbers on the blockchain. Entropy's rapid response time allows 
+developers to build applications such as NFT mints and games with responsive UX. 
+Entropy also provides strong security guarantees to ensure that both 
+users and application developers can trust that the results are random.
+
+Pyth Entropy is currently available on Zetachain. [Learn more] (https://docs.pyth.network/entropy/generate-random-numbers) on how to generate a random number through Entropy.
+
+Pyth contract addresses:
+
+| Network      | Address                                                                                                                                    |
+| :----------- | :----------------------------------------------------------------------------------------------------------------------------------------- |
+| Mainnet      | [0x36825bf3Fbdf5a29E2d5148bfe7Dcf7B5639e320](https://zetachain.blockscout.com/address/0x36825bf3Fbdf5a29E2d5148bfe7Dcf7B5639e320)          |
+| Testnet      | [0x4374e5a8b9C22271E9EB878A2AA31DE97DF15DAF](https://explorer.zetachain.com/address/0x4374e5a8b9C22271E9EB878A2AA31DE97DF15DAF)            |
```

Comment:
```suggestion
| Testnet      | [0x4374e5a8b9C22271E9EB878A2AA31DE97DF15DAF](https://zetachain-athens-3.blockscout.com/address/0x4374e5a8b9C22271E9EB878A2AA31DE97DF15DAF)            |
```

I think it's better if we use the same explorer for both. Blockscout is currently slightly better for EVM side of things, so I'd go with that.